services/cognito-presignup/template.yaml

```diff
@@ -0,0 +1,19 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: 'AWS::Serverless-2016-10-31'
+Description: An AWS Serverless Specification template describing your function.
+Resources:
+  cognitoPreSignUpTest:
+    Type: 'AWS::Serverless::Function'
+    Properties:
+      Handler: lambda_function.lambda_handler
+      Runtime: python3.8
+      CodeUri: cognitoPreSignUpTest-ca6f9264-22c4-4134-a69a-aafbb5ba31dd.zip
```

Comment:
Uploading to an S3 location as an artifact is the ideal solution, but this will do.